src/main/java/frc/robot/Telemetry.java

```diff
@@ -25,11 +26,14 @@ public class Telemetry {
    */
   public Telemetry(double maxSpeed) {
     MaxSpeed = maxSpeed;
```

Comment:
I know this is pre-exiating code, but idiomatic Java usually uses `lowerCaseFirst` naming for variables, and `UpperCaseFirst` for classes. Please update this variable accordingly, and then you can differentiate it from the parameter of the same name using `this.maxSpeed`. E.g.:

```suggestion
    this.maxSpeed = maxSpeed;
```

Reply:
This gives me an error when changing it to this. I leave that for another pull request.